settings.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from dotenv import load_dotenv
+from pydantic.v1 import BaseSettings
+
+
+load_dotenv()
+
+
+class Settings(BaseSettings):
+    DATABASE_URL: str | None = "sqlite+aiosqlite:///./city_temperature.db"
```

Comment:
The use of 'str | None' for the 'DATABASE_URL' field requires Python 3.10 or later. Ensure that your environment supports this syntax or use 'Optional[str]' from the 'typing' module for compatibility with earlier versions.